views/admin/sendmail/index.tt

```diff
@@ -12,45 +12,60 @@
 %]
 
 [%- sections.title='.title' | ml -%]
+[%- sections.head = BLOCK %]
+    <style type="text/css">
+        .error { color: #f04124; }
+    </style>
+[% END %]
 
 <p>[% '.intro' | ml %]</p>
 
+[%- INCLUDE components/errors.tt errors = errors -%]
+
 <form method="POST">
 
 [%- IF has_menu -%]
     [% dw.form_auth %]<p>
-    [%- label_a = ".select.account.label" | ml;
-        form.select( label = label_a, name = "account", id = "account",
+    [%- form.select( label = dw.ml( ".select.account.label" ),
```

Comment:
ps you can remove the "id=" these days, unless you need it to target JS.

(the old reason we wanted it was for accessibility, so we could map the label to the textbox; but there's something in place that does that automatically now. one less thing to have to remember!)